urllib3/poolmanager.py

```diff
@@ -112,6 +113,8 @@ class PoolManager(RequestMethods):
 
     def __init__(self, num_pools=10, headers=None, **connection_pool_kw):
         RequestMethods.__init__(self, headers)
+        if "transport_security_manager" not in connection_pool_kw:
+            connection_pool_kw["transport_security_manager"] = TransportSecurityManager()
```

Comment:
I've exposed `transport_security_manager` as a `PoolManager` attribute.

I'm not entirely sure what needs to be done to key connection pools off TSMs. It's already the case that a connection pool can only be associated with one TSM; a pool manager is also expected to only be associated with one TSM. Could you explain what I need to do?